src/Augmentation/data/AugmentationCreator.tsx

```diff
@@ -526,9 +526,9 @@ export const initGeneralAugmentations = (): Augmentation[] => [
     moneyCost: 1.375e9,
     info:
       "A cranial implant that greatly assists in the user's ability to analyze social situations " +
-      "and interactions. The system uses a wide variety of factors such as facial expressions, body " +
-      "language, and the voice tone, and inflection to determine the best course of action during social" +
-      "situations. The implant also uses deep learning software to continuously learn new behavior" +
+      "and interactions. The system uses a wide variety of factors such as facial expression, body " +
+      "language, voice tone, and inflection to determine the best course of action during social " +
+      "situations. The implant also uses deep learning software to continuously learn new behavior " +
```

Comment:
in game renders the words 'social', 'situation' and 'behavior', 'patterns' as 1 word.